lib/matplotlib/type1font.py

```diff
@@ -35,9 +34,213 @@
 
 _log = logging.getLogger(__name__)
 
-# token types
-_TokenType = enum.Enum('_TokenType',
-                       'whitespace name string delimiter number')
+
+class _Token:
+    """
+    A token in a PostScript stream
+
+    Attributes
+    ----------
+    pos : int
+        position, i.e. offset from the beginning of the data
+
+    value : bytes
+        the text of the token
+    """
+    __slots__ = ('pos', 'value')
+    pos: int
+    value: bytes
```

Comment:
We're still not using type annotations in the library right now so would rather remove these (and below), although this specific part is so far away from the plotting logic that I guess it's fine if you do want to keep them.